tests/build_memory_index.cpp

```diff
@@ -85,6 +88,12 @@ int main(int argc, char** argv) {
         po::value<uint32_t>(&num_threads)->default_value(omp_get_num_procs()),
         "Number of threads used for building index (defaults to "
         "omp_get_num_procs())");
+    desc.add_options()("num_pq_bytes",
+                       po::value<uint32_t>(&build_PQ_bytes)->default_value(0),
+                       "Number of PQ bytes to compress vectors to");
```

Comment:
nit: should this description clarifies that the "PQ bytes" refers to size of compressed vectors used for distance comparisons during index build (i.e., not to confuse with the size of compressed vectors during serving time, which is derived from -B option)?

Reply:
fixed